packages/api-tests/reporter.js

```diff
@@ -0,0 +1,62 @@
+var mocha = require('mocha');
+module.exports = MyReporter;
+const testContainer = process.env.MOCHA_CONTAINER;
+var fs = require('fs');
+var path = require('path');
+
+let jsonOutput = [];
+const pattern = /(#[\w.()]+[\s]?)+/;
```

Comment:
Maybe give the variable a more descriptive name (indicating what it is a pattern for)? 

Is the whitespace expression (`[\s]?`) needed?